ac/ac-induction/src/TestPart/Test.jsx

```diff
@@ -66,13 +66,8 @@ export default ({
       <ExLine />
       <ExContainer style={{ padding: '20px' }}>
         <TestCorrectionDiv style={{ justifyContent: 'space-evenly' }}>
-          <h3>This image corresponds to an example of the concept </h3>
-          <Switch
-            tmpList={tmpList}
-            feedback={feedback}
-            data={data}
-            dataFn={dataFn}
-          />
+          <h3>This example is a {title}</h3>
```

Comment:
I think it was the same before, but it really looks like an affirmation instead of a sentence linked to the toggle button. This might confuse the student

Reply:
I don't think it is a problem. I believe people are used to the type of question: Affirmation + True/False ? But we should keep that in mind for our pre-experiment